Board.py

```diff
@@ -9,7 +9,7 @@ def create_board(self) -> list[list[str]]:
                      ['|_|','_|','_|','_|','_|','_|'],
                      ['|_|','_|','_|','_|','_|','_|'],
                      ['|_|','_|','_|','_|','_|','_|'],
-                     ['|1|''2|''3|''4|''5|''6|']]
+                     ['|1|', '2|', '3|', '4|', '5|', '6|']]
```

Comment:
```suggestion
                     ['|1|','2|','3|','4|','5|','6|']]
```
Remove spaces since you didn't use then in the rows above.